thorlcr/activities/lookupjoin/thlookupjoinslave.cpp

```diff
@@ -943,6 +943,7 @@ class CInMemJoinBase : public CSlaveActivity, public CAllOrLookupHelper<HELPER>,
     IArrayOf<IRowStream> gatheredRHSNodeStreams;
     bool rhsConstant = false;
     bool rhsStartedBefore = false;
+    bool lhsLookAheadConfigured = false;
```

Comment:
This flag and the associated code should be in a base class - it has been cut and pasted into too many files.